actions/v2/admin/internal/mapping/user.go

```diff
@@ -1,24 +1,24 @@
 package mapping
 
 import (
-	"github.com/bitcoin-sv/spv-wallet/engine/utils"
 	"github.com/bitcoin-sv/spv-wallet/engine/v2/users/usersmodels"
 	"github.com/bitcoin-sv/spv-wallet/models/response/adminresponse"
+	"github.com/samber/lo"
 )
 
-// CreatedUserResponse maps a user to a response
-func CreatedUserResponse(u *usersmodels.User) adminresponse.User {
+// UserToResponse maps a user to a response
+func UserToResponse(u *usersmodels.User) adminresponse.User {
 	return adminresponse.User{
 		ID:        u.ID,
 		CreatedAt: u.CreatedAt,
 		UpdatedAt: u.UpdatedAt,
 		PublicKey: u.PublicKey,
-		Paymails:  utils.MapSlice(u.Paymails, UsersPaymailResponse),
+		Paymails:  lo.Map(u.Paymails, UsersPaymailToResponse),
 	}
 }
 
-// UsersPaymailResponse maps a user's paymail to a response
-func UsersPaymailResponse(p *usersmodels.Paymail) adminresponse.Paymail {
+// UsersPaymailToResponse maps a user's paymail to a response
+func UsersPaymailToResponse(p *usersmodels.Paymail, _ int) adminresponse.Paymail {
```

Comment:
maybe some util function like that:
```go
func NoIndex[T any, R any](mapper func(T) R) func(T, int) R {
	return func(item T, _ int) R {
		return mapper(item)
	}
}
```
Would allow to make a single item mapper without index argument:
```suggestion
		Paymails:  lo.Map(u.Paymails, NoIndex(UsersPaymailToResponse)),
	}
}

// UsersPaymailToResponse maps a user's paymail to a response
func UsersPaymailToResponse(p *usersmodels.Paymail) adminresponse.Paymail {
```

Reply:
I don't like the `NoIndex` name - for people who are not very into this mappin feature - this might be misunderstanding.

Reply:
Sure! sounds good